patient_registration.md

```diff
@@ -23,6 +23,28 @@
 3.9.\* Confirm: Shows all the bio-data collected about the patient.![Confirm](images/confirmation.png)  
 3.10. Saving The data: Click confirm to save, if not click cancel to edit. You should see a patient registration summary once saving is successful as below ![](/assets/poc_patient_summary.PNG)
 
+###Editing Patient Registration Information
+
+###Editing  Patient Registration
+
+1.Login and on the home page click the Find Patient Link circled below
+
+![Register Patient Link](/assets/poc_find_patient_link.png)
+
+2.In the Search Bar, type any patient identifiers to search for the patient, the record of the patient appears as shown below.
+
+![Patient Search Record](/assets/poc_patient_record_search.png)
+
+3.Click on the last action that has Edit Demographics as shown below;
```

Comment:
@solemabrothers remove this duplicated heading

Reply:
@ssmusoke I dont see any duplicate here.